html/semantics/forms/form-submission-0/constructing-form-data-set.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<head>
+<meta charset="utf-8">
+<link rel="help" href="https://html.spec.whatwg.org/multipage/form-control-infrastructure.html#constructing-the-form-data-set">
+<link rel="help" href="https://fetch.spec.whatwg.org/#ref-for-utf-8">
```

Comment:
I changed this to https://fetch.spec.whatwg.org/#concept-bodyinit-extract, and added https://xhr.spec.whatwg.org/#dom-formdata too.

I want to mention Fetch spec here because it explains ```_charset_``` value for ```new FormData(form)``` is ```"UTF-8"```.